README.md

```diff
@@ -46,11 +46,14 @@ These are some quick hints, especially for those coming from a ROS1 control back
 * <ros2_control> tags in the URDF must be compatible with the controller's configuration.
 * PLUGINLIB_EXPORT_CLASS macro is required when implementing an interface.
 
-# Test of the Scenario Before the First Release
+# Testing from the sources
 * Checkout [ros-controls/ros2_control](https://github.com/ros-controls/ros2_control) to get the core.
 * Checkout [ros-controls/ros2_controllers](https://github.com/ros-controls/ros2_controllers) to get all the controllers.
 * Checkout [ros-controls/ros2_control_demos](https://github.com/ros-controls/ros2_control_demos) to get example hardware and robot launch files.
 
+**NOTE**: `ros2_control` and `ros2_controllers` packages are released for foxy and can be installed using package manager.
+  Still, it is recommended to use source version since there might be some not-yet-released changes.
```

Comment:
```suggestion
  For daily use it is recommended to use the released version but there may always be some not-yet-released changes that are required to build the demos.
```
I'd not advise anyone to check things out from source. We are trying to advocate for a stable ROS distribution, not the wild west! :cowboy_hat_face: